scanner/e2etests/helmchart/templates/scanner-deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         image: "{{ ( printf "%s/%s:%s" .Values.image.repository .Values.app.scanner.name .Values.image.tag ) }}"
         imagePullPolicy: IfNotPresent
         command:
-          - /entrypoint.sh
+          - entrypoint.sh
```

Comment:
The similar should also be done in the actual deployment Helm charts, if they exist already.

This entire change is for Scanner V4, right?

Reply:
It's still in review, and I'll be sure to check this in my next pass over the PR: https://github.com/stackrox/stackrox/pull/8445